[Cookbook] Additional content

LukeSavefrogs opened the pull request:
Hi @pythonexcels! 

Thank you for this amazing guide! I used it along with the [official VBA documentation](https://learn.microsoft.com/en-us/office/vba/api/overview/excel/object-model) to move my first steps into the Excel COM world. 

I wanted to add some information to this already amazing website, so i forked it, added some content and improved a bit the formatting (added TOC, moved recipes into a separate section and so on).

I hope you'll like my changes and merge them to the original code, so that can be used by others as well.

LukeSavefrogs commented:
One of the changes i made was adding syntax highlighting to code snippets.

![Screenshot](https://user-images.githubusercontent.com/33452387/198824438-45ebf7d2-93bd-4003-8532-d1a3158f3b65.png)

This way is easier to follow the tutorials.

pythonexcels commented:
Thanks @LukeSavefrogs for the suggestions. I implemented syntax highlighting and shorted some link references as suggested in https://github.com/pythonexcels/pythonexcels.github.io/pull/7. I didn't want to change `wb` to `workbook` and `ws` to `worksheet` since the wb/ws convention is used throughout the guide and I don't have a great testing setup ATM. Thanks again!

LukeSavefrogs commented:
> Thanks @LukeSavefrogs for the suggestions. I implemented syntax highlighting and shorted some link references as suggested in #7. I didn't want to change `wb` to `workbook` and `ws` to `worksheet` since the wb/ws convention is used throughout the guide and I don't have a great testing setup ATM. Thanks again!

No problem, the renaming was just for clarification. 
What about the other recipes i added ("[Set zoom for all worksheets](https://github.com/pythonexcels/pythonexcels.github.io/pull/6/commits/6296d63ea57ad60e3230572940a01549bf7b3f3c)"/"[Close without saving](https://github.com/pythonexcels/pythonexcels.github.io/pull/6/commits/c968231137aaeda87cc95f1ca88c91a7ab4b356a)")? These could be useful 👍🏻 

Thank you anyway for taking into account my suggestions 😄
